README.md

```diff
@@ -771,8 +780,8 @@ Which produces unsettled trades like so:
 Before you are able to fill orders or submit new orders, you need to approve the SportX contracts for trading. You can do this by simply calling:
 
 ```typescript
-const result = await sportX.approveSportXContracts(tokenAddress);
-console.log(result);
+const approvalTxn = await sportX.approveSportXContracts(tokenAddress, mnemonic);
```

Comment:
Don't think `mnemonic` should be there - doesn't seem it matches the signature

Reply:
forgot to remove it from README. fixed